indico/modules/events/templates/header.html

```diff
@@ -38,15 +38,20 @@
        title="{% trans %}Change theme{% endtrans %}"></a>
     <ul class="dropdown">
         {% set endpoint = 'timetable.timetable' if event.type == 'conference' else 'events.display' %}
-        {% for id, data in themes.items()|sort(attribute='1.name')
-           if event.can_manage(session.user) or data.user_visible %}
-            <li>
-                {% if id == theme %}
-                    <a class="disabled"><strong>{{ data.name }}</strong></a>
-                {% else %}
-                    <a href="{{ url_for(endpoint, event, view=id) }}">{{ data.name }}</a>
-                {% endif %}
-            </li>
+        {% for user_visible, themes in themes.items()|sort(attribute='1.name')|groupby('1.user_visible')|reverse
```

Comment:
I think shadowing `themes` with `themes` is rather ugly. I'd try to find a different name (which might be hard) or use the grouper/list style of using groupby as shown in the docs (didn't even know that existed, just saw it when looking if there's a `reversed` attribute for groupby!): http://jinja.pocoo.org/docs/2.10/templates/#groupby